Dockerfile

```diff
@@ -66,18 +57,30 @@ RUN apt-get install -y libzmq3-dev python-pip default-jdk && \
     touch /root/.jupyter/jupyter_nbconvert_config.py && touch /root/.jupyter/migrated
 
 # Tensorflow and Keras
-RUN pip install virtualenv && R -e 'keras::install_keras()' 
+ENV WORKON_HOME=/usr/local/share/.virtualenvs
```

Comment:
Can you add a comment to explain how this new `WORKON_HOME` env variable is used?

Reply:
Done.